src/Models/PostModel.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+require_once 'src/Entities/Post.php';
+
+class PostModel {
+
+    private PDO $db;
+    public function __construct(PDO $db)
+    {
+        $this->db = $db;
+    }
+
+    public function getAllPosts()
+    {
+        $query = $this->db->prepare('SELECT `id`,`title`, `content`,`author-name`, `date-time`, `user-id` 
+        FROM `posts` ORDER BY `date-time` DESC');
+        $query->execute();
+        $data = $query->fetchAll();
+
+        return $data;
+    }
+
+    public function displayAllPosts() :void
```

Comment:
This should not be here, a model should not contain anything other than DB queries. 

You should move this method into a new class. Classes that display stuff can be called ViewHelpers - so in this case I would recommend calling it PostsViewHelper